### PR TITLE
Examples for creating signatures and attendees via API

### PR DESCRIPTION
**Pull request by anero**
## Example for creating a Signature

image

## Example for creating an Attendee
This example already existed, I just updated to make it clearer how the email opt-in type is set

image

### CODE REVIEW

**Review by woodhull — APPROVED** [review]
This is definitely better. I think we could use a paragraph somewhere more than we already have where we explain email optin in types, their external IDs, etc -- that we may want to link to from the table of signature fields

File: source/includes/authenticated_api/_signatures.md.erb [hunk 1]
```diff
@@ -132,6 +132,22 @@ The JSON response has a single "signature" object, which may include the followi
...
+> POST /api/v1/petitions/no-taxes-on-tea/signatures/
+
+```json
+{
+  "signature": {
+    "first_name": "Foo",
+    "last_name": "Bar",
+    "email": "foo@bar.com",
+    "postcode": "10010",
+    "phone_number": "123-456",
```
Comment (lavaturtle) [inline source/includes/authenticated_api/_signatures.md.erb]: nitpick: this isn't a valid US phone number
(We could use `555-555-5555` instead)

**Review by lavaturtle — APPROVED** [review]
:+1:

**Comment by anero** [comment]
> This is definitely better. I think we could use a paragraph somewhere more than we already have where we explain email optin in types, their external IDs, etc -- that we may want to link to from the table of signature fields

We already have documentation on each of the parameters, how to get their values using other API endpoints, and whether they're required or not: https://developers.controlshiftlabs.com/#authenticated-rest-api-signatures-create. What we're missing were examples on the actual payload. I decided to include a short version of the payload, with the minimum set of parameters to keep it simple. I am hoping that people will figure out how to send the other parameters from the complete list and the example.

**Pull request merged**